python/test/openmldb_client_test.py

```diff
@@ -314,10 +316,48 @@ def test_parameterized_query(self):
       (1008, '2021-01-02', 'province3', 'city9', 9, 1590738998000),
       ]
     self.check_result(rs, expectRows)
- 
 
+# test sqlalchemy Table-object-based API in pytest style
+class TestSqlalchemyAPI:
+
```

Comment:
define object members in `__init__()`.

Reply:
In Pytest's conventions, `test prefixed test functions or methods inside Test prefixed test classes (without an __init__ method)`.
it does not support the `Test` class with a `__init__` function, otherwise, it raise error like: ` cannot collect test class 'TestSqlalchemyAPI' because it has a __init__ constructor.`